Lib/distutils/tests/test_register.py

```diff
@@ -291,6 +291,7 @@ def test_check_metadata_deprecated(self):
         cmd = self._get_cmd()
         with check_warnings() as w:
             warnings.simplefilter("always")
+            warnings.filterwarnings("ignore", ".*OptionParser class will be replaced.*")
```

Comment:
Add line for second Deprecation message added to issue report.
```suggestion
            warnings.filterwarnings("ignore", ".*OptionParser class will be replaced.*")
            warnings.filterwarnings("ignore", ".*Option class will be removed.*")
```